assignment/tool/src/webapp/js/studentViewSubmission.js

```diff
@@ -1,4 +1,8 @@
 var ASN_SVS = ASN_SVS || {};
+var ASN = ASN || {};
+var ASN_TS_API = ASN_TS_API || {};
```

Comment:
Try and use const wherever possible. Makes our js more reliable.

Reply:
Those 3 are the only places where var cannot be changed to const, because those values may already exist